gnd/build.gradle

```diff
@@ -103,6 +107,42 @@ android {
     hilt {
         enableTransformForLocalTests = true
     }
+    DomainObjectSet<BaseVariant> variants = getApplicationVariants()
+    variants.addAll(getTestVariants())
+    variants.addAll(getUnitTestVariants())
+    variants.configureEach { variant ->
+        variant.getJavaCompileProvider().configure {
+            options.errorprone {
+                check("NullAway", CheckSeverity.ERROR)
+                option("NullAway:AnnotatedPackages", "com.google.android.gnd")
+
+                // Treat generated classes from Hilt and SafeArgs as unannotated.
+                option("NullAway:TreatGeneratedAsUnannotated", true)
+
+                // Ignore generated classes from databinding, as well as Room and Firebase classes.
+                option("NullAway:UnannotatedSubPackages",
+                        "com.google.android.gnd.databinding," +
+                        "com.google.android.gnd.persistence.local.room," +
+                        "com.google.android.gnd.persistence.remote.firestore," +
+                        "com.google.android.gnd.persistence.remote.firestore.schema")
+
+                // TODO: Figure out a better way of excluding classes generated by the SafeArgs
+                //  plugin. Suggestions: patch NullAway to ignore methods annotated with
+                //  @SuppressWarnings("unchecked") or patch SafeArgs to annotate the generated
+                //  classes with @Generated
+                option("NullAway:ExcludedClasses",
```

Comment:
Is it possible to exclude patterns instead of full class/package name?

Reply:
Annoyingly no. It is possible to use regex to exclude package names but not for class names.  

Config options are here: https://github.com/uber/NullAway/wiki/Configuration